frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1010,6 +1028,37 @@ pub mod pallet {
 			});
 			Ok(())
 		}
+
+		/// Trigger the governance fallback.
```

Comment:
Is `OnChainSeqPhragmen` the only current option that we offer out of the box that would work here? If so might be good to mention that

Reply:
Yes, but anyone can write a custom one as well.